### PR TITLE
Develop rc2

### PR DESCRIPTION
**Pull request by jonni-larjomaa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,16 +103,19 @@ In order to charge a card given in the Form API, the corresponding transaction i
...
 ```php
 
 use Solinor\PaymentHighway\Model\Security\SecureSigner
 
 $secureSigner = new SecureSigner(signatureKeyId, signatureSecret);
 
-if ( ! secureSigner->validateFormRedirect(requestParams)) {
-throw new Exception("Invalid signature!");
+try{
+    $secureSigner->validateFormRedirect($params)) { // request params as [ key => value] array
```
Comment (Swie) [inline README.md]: Maybe clearer with  "redirected get parameters" instead of request params

**Comment by jonni-larjomaa** [comment]
RC2

File: README.md [hunk 2]
```diff
@@ -132,16 +136,18 @@ $signatureSecret = "testSecret";
...
 
-$paymentApi = new PaymentApi($serviceUrl, $signatureKeyId, $signatureSecret, $account, $merchant)
+$paymentApi = new PaymentApi($serviceUrl, $signatureKeyId, $signatureSecret, $account, $merchant);
 ```
         
 Example Commit Form Transaction
 ```php
-$transactionId = ""; // get sph-transaction-id as a GET parameter
-$amount = "1999";
+$transactionId = 54321; // get sph-transaction-id as a GET parameter
```
Comment (Swie) [inline README.md]: Transaction ID is a UUIDV4 instead of an int. Example could be one as well

File: src/Security/SecureSigner.php [hunk 3]
```diff
@@ -47,16 +51,47 @@ public function createSignature($method, $uri, array $nameValuePairs, $body = ""
...
+     * @param $method
+     * @param $uri
+     * @param array $nameValuePairs either headers or request params in format [ key => value ]
+     * @param string $body
+     * @throws Exception
+     */
+    public function validateSignature($method, $uri, array $nameValuePairs, $body = "")
+    {
+
+        $expectedSignature = $nameValuePairs["signature"];
```
Comment (Swie) [inline src/Security/SecureSigner.php]: I think expected and actual signatures should be swapped, as we expect the one we calculate and actual was the one given :) In java lib the term "receivedSignature" is used

File: src/Model/Request/Transaction.php [hunk 4]
```diff
@@ -18,14 +18,14 @@ class Transaction implements \JsonSerializable
...
-     * @param Card|Token $request
      * @param int $amount
      * @param string $currency
+     * @param Card|Token $request
      * @param bool $blocking
      * @param string $orderId
      * @throws Exception
      */
-    public function __construct( $request, $amount, $currency, $blocking = true, $orderId = null )
+    public function __construct( $amount, $currency, $request = null, $blocking = true, $orderId = null )
```
Comment (Swie) [inline src/Model/Request/Transaction.php]: Could roll back this change when the commit function call no longer requires a transaction object

**Pull request merged**